src/xbps/advanced-usage.md

```diff
@@ -56,3 +56,14 @@ ignorepkg=sudo
 
 You will then be able to remove the `sudo` package using
 [xbps-remove(1)](https://man.voidlinux.org/xbps-remove.1).
+
+## Virtual Packages
+
+Virtual packages can be created with
+[xbps.d(5)](https://man.voidlinux.org/xbps.d.5) `virtualpkg` entries. For
+example, to make the `linux` package refer to the `linux5.6` package, create an
```

Comment:
I believe "refer" isn't the best wording. I really like the man page explanation. It uses "resolve", in this case, but I would copy the whole phrase, really: "Any request to the virtual package will be resolved to the real package."